### PR TITLE
Add reproducible for powpeg-node 6.4.0.0

### PR DESCRIPTION
**Pull request by nathanieliov**

### CODE REVIEW

File: powpeg-node/ARROWHEAD-6.4.0.0/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    git fetch --depth 1 origin tag "$gitrev" && \
+    git checkout "$gitrev"
+
+RUN gpg --keyserver https://secchannel.rsk.co/SUPPORT.asc --recv-keys A6DBEAC640C5A14B && \
+    gpg --verify --output SHA256SUMS SHA256SUMS.asc && \
+    sha256sum --check SHA256SUMS && \
+    ./configure.sh && \
+    ./gradlew --no-daemon clean build -x test
+
+FROM openjdk:8-jre-slim-buster as runner
```
Comment (marcos-iov) [inline powpeg-node/ARROWHEAD-6.4.0.0/Dockerfile:20]: jdk8

**Comment by jeremy-then** [comment]
6b3521d6a1905e72b01730c8be82423da22e95686777806392a53c8ae8f622bc  federate-node-ARROWHEAD-6.4.0.0-all.jar
d895cc6df962565e73d57ab439e5822219fd42935cb46ad8a2216174c3c60f9d  federate-node-ARROWHEAD-6.4.0.0-sources.jar
7a2a85d275c2c2abf8f3a85aaf420a8ef512670d2f8e42c281c7821094ffdcdc  federate-node-ARROWHEAD-6.4.0.0.jar

**Review by jeremy-then — APPROVED** [review]
LGTM.

File: powpeg-node/ARROWHEAD-6.4.0.0/Dockerfile (hunk 1 above)
Comment (nathanieliov) [reply powpeg-node/ARROWHEAD-6.4.0.0/Dockerfile]: Fixed

**Review by marcos-iov — APPROVED** [review]
```
6b3521d6a1905e72b01730c8be82423da22e95686777806392a53c8ae8f622bc  federate-node-ARROWHEAD-6.4.0.0-all.jar
7a2a85d275c2c2abf8f3a85aaf420a8ef512670d2f8e42c281c7821094ffdcdc  federate-node-ARROWHEAD-6.4.0.0.jar
d895cc6df962565e73d57ab439e5822219fd42935cb46ad8a2216174c3c60f9d  federate-node-ARROWHEAD-6.4.0.0-sources.jar
```

**Comment by julia-zack** [comment]
```
6b3521d6a1905e72b01730c8be82423da22e95686777806392a53c8ae8f622bc  federate-node-ARROWHEAD-6.4.0.0-all.jar
7a2a85d275c2c2abf8f3a85aaf420a8ef512670d2f8e42c281c7821094ffdcdc  federate-node-ARROWHEAD-6.4.0.0.jar
d895cc6df962565e73d57ab439e5822219fd42935cb46ad8a2216174c3c60f9d  federate-node-ARROWHEAD-6.4.0.0-sources.jar
```

**Review by julia-zack — APPROVED** [review]

**Pull request merged**